src/Mod/Sketcher/Gui/Utils.h

```diff
@@ -119,6 +119,9 @@ inline bool isEdge(int GeoId, Sketcher::PointPos PosId)
 // Return counter-clockwise angle from horizontal out of p1 to p2 in radians.
 double GetPointAngle (const Base::Vector2d &p1, const Base::Vector2d &p2);
 
+// Set the two points on circles at minimal distance
+bool GetCirclesMinimalDistance(const Part::GeomCircle *circle1, const Part::GeomCircle *circle2, Base::Vector3d *point1, Base::Vector3d *point2);

```

Comment:
It is unconventional to pass stack parameters by pointer (convention is by reference). A developer reading the code tends to think that point1 and point2 are dynamically allocated in the heap. It is thus a little bit misleading.

```..., Base::Vector3d & point1, Base::Vector3d& point2);```

is what we are looking for.